### PR TITLE
Flask cheatsheet

### PR DESCRIPTION
**Pull request by marksteve**
WIP

### CODE REVIEW

File: flask.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+  - WIP
+  - Python
+---
+
+{% raw %}
+
+Flask
+-----
+
+## Configuration
```
Comment (rstacruz) [inline flask.md]: an `h2` followed by an `h2` seems strange, but you probably did this to make the "Configuration" heading visible. may I suggest filling in the "Flask" heading with a super-quick getting started guide / hello world?

eg: https://devhints.io/fastify

File: flask.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+### Setting Configuration
+
+```python
+app = Flask(__name__)
+app.config['DEBUG'] = True
+app.config.from_object('yourapplication.default_settings')
+app.config.from_envvar('YOURAPPLICATION_SETTINGS')
+```
+
+### Builtin Configuration Values
```
Comment (rstacruz) [inline flask.md]: I'd keep this in "Sentence case" rather than "Title Case" - just to be consistent with the rest of the site :)

File: flask.md [hunk 3]
```diff
@@ -0,0 +1,241 @@
+---
+title: Flask
+layout: 2017/sheet
+category: Flask  # could be split later on
```
Comment (rstacruz) [inline flask.md]: this category doesn't exist in `_data/categories.yml`, maybe start with a `Python libraries` category. Btw, I can help on this stuff later!

File: flask.md (hunk 2 above)
Comment (marksteve) [reply flask.md]: Sure thing! I was mostly copy-pasting stuff from the documentation haha

File: flask.md (hunk 1 above)
Comment (marksteve) [reply flask.md]: Will do!

**Pull request closed without merging**